script.js

```diff
@@ -0,0 +1,53 @@
+// var choices = [1, 2, 3, 4, 5, 6];
+//maybe change "choices" to "Prompts" and the oucomes of prompts are the parameteres for moving on
+
+// var bluePill = false;
+// var redPill = true;
+
+
+
+var ready = prompt("Wake up, Neo. Are you ready?");
+
+ if (ready != null) {
+    confirm("Great. Follow the white rabbit");
+
+  }
+  else {
+  confirm("..but you have to be, Neo. You are the One. The Matrix has you. Follow the white rabbit. 'Knock, Knock")
+};
+
+  var meetMorpheus = prompt("After a bit of trouble, you meet Morpheus." + " " + "Morpheus says: 'Welcome, Neo. Do you know who I am?''");
+
+    if( meetMorpheus != null || "no" || "No") {
```

Comment:
this will always evaulate to true `meetMorpheus != null || "no" || "No"` Basically what your sayins is `meetMorpheus !- null` or `"no"` or `"No"`. Because strings are truthy in javascript, this condition will ALWAYS evaluate to true.